__tests__/helpers/database.js

```diff
@@ -0,0 +1,36 @@
+import {
+  internet, lorem, name,
+} from 'faker';
+import _ from 'lodash';
+
+export default (app) => ({
+  insert: {
+    status: (data = {}) => app.objection.models.status.query()
+      .insert(_.merge({ name: name.title() }, data)),
+    label: (data = {}) => app.objection.models.label.query()
+      .insert(_.merge({ name: name.title() }, data)),
+    user: (data = {}) => app.objection.models.user.query()
+      .insert(_.merge({
+        firstName: name.firstName(),
+        lastName: name.lastName(),
+        email: internet.email(),
+        password: internet.password(),
+      }, data)),
+    task: (data = {}) => app.objection.models.task.query()
+      .insert(_.merge({
+        name: name.title(),
+        description: lorem.paragraph(),
+      }, data)),
+  },
+  find: {
+    statuses: () => app.objection.models.status.query(),
+    users: () => app.objection.models.user.query(),
+    labels: () => app.objection.models.label.query(),
+    tasks: () => app.objection.models.task.query(),
+  },
+  clear: async () => {
+    // rollback and migrate to prune database
+    await app.objection.knex.migrate.rollback();
+    await app.objection.knex.migrate.latest();
```

Comment:
i guess factories and test teardown are not really related

Reply:
ith not a factory, its a db client for tests

Reply:
pr name says opposite

Reply:
thats because on issue name, in which there is a last comment which states on what we agreed to do in general with test data :)

Reply:
we agreed on use common practices. 
factories should not be responsible for data lookups. And most probably you will find yourself writing a lot of code building factories when it will come to relations.

Decide for yourself, move build/create to factories. Name the rest "databaseHelpers"